### PR TITLE
pin Mux Player to ^1 and add Mux Data metadata

### PR DESCRIPTION
**Pull request by dylanjha**

### CODE REVIEW

File: components/CourseViewer.tsx [hunk 1]
```diff
@@ -66,6 +66,11 @@ const CourseViewer = ({ course, lessonProgress = [], setLessonProgress }: Props)
             playbackId={playbackId}
             placeholder={placeholder}
             onEnded={markLessonCompleted}
+            metadata={{
```
Comment (cjpillsbury) [inline components/CourseViewer.tsx:69]: praise: 😎

File: components/CourseViewer.tsx [hunk 2]
```diff
@@ -66,6 +66,11 @@ const CourseViewer = ({ course, lessonProgress = [], setLessonProgress }: Props)
...
+              video_series: activeLesson.courseId,
+              video_title: activeLesson.name,
+              player_name: "Video Course Starter Kit",
```
Comment (cjpillsbury) [inline components/CourseViewer.tsx:72]: question(non-blocking): Any reason you included "Mux Player" in the name for stream.new but are not including it here?

File: package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@mux/blurhash": "^0.1.2",
     "@mux/mux-node": "^6.3.0",
-    "@mux/mux-player-react": "^1.2",
+    "@mux/mux-player-react": "^1",
```
Comment (cjpillsbury) [inline package.json:15]: bold.

**Review by cjpillsbury — APPROVED** [review]
One (non-blocking) question. Otherwise 👌

File: components/CourseViewer.tsx (hunk 2 above)
Comment (dylanjha) [reply components/CourseViewer.tsx:72]: No reason, I think I'm going to change it on stream.new side actually, cause Mux Player will come through the `player_software_name`. Good call

**Pull request merged**